### PR TITLE
Added except blocks for CaseError thrown when getting case properties

### PR DESCRIPTION
**Pull request by orangejenny**
https://dimagi-dev.atlassian.net/browse/HI-715

The `CaseError` is already captured in the usual place it's thrown, in [form view](https://github.com/dimagi/commcare-hq/blob/8dccb4cb5f3114732e3386e9ed3ab144e99d31b0/corehq/apps/app_manager/views/forms.py#L683-L684) for the problematic form.

But it can also pop up when getting case properties in an app, because getting the case properties for a module that uses the visit scheduler [calls add_stuff_to_xform](https://github.com/dimagi/commcare-hq/blob/8dccb4cb5f3114732e3386e9ed3ab144e99d31b0/corehq/apps/app_manager/models.py#L2684).

Feature flag: visit scheduler

### CODE REVIEW

File: corehq/apps/app_manager/views/forms.py [hunk 1]
```diff
@@ -721,14 +723,24 @@ def get_form_view_context_and_template(request, domain, form, langs, current_lan
         valid_index_names.append(USERCASE_PREFIX[0:-1])     # strip trailing slash
 
     form_has_schedule = isinstance(form, AdvancedForm) and module.has_schedule
+
+    try:
+        case_properties_map = get_all_case_properties(app)
+        usercase_properties_map = get_usercase_properties(app)
+    except CaseError as e:
+        case_properties_map = {}
+        usercase_properties_map = {}
```
Comment (esoergel) [inline corehq/apps/app_manager/views/forms.py]: Is it likely that only one of these calls will fail?  Would it be useful to let the other go through, or is that unimportant?

**Review by esoergel — APPROVED** [review]

File: corehq/apps/app_manager/views/forms.py (hunk 1 above)
Comment (orangejenny) [reply corehq/apps/app_manager/views/forms.py]: I think that if either of them error, your app needs to be fixed before you can meaningfully use the page, and you'll need to refresh after fixing it to correctly populate the properties map, so I'm not worried about making the page all that useful while you're in the error state.

File: corehq/apps/app_manager/views/forms.py [hunk 2]
```diff
@@ -721,14 +723,24 @@ def get_form_view_context_and_template(request, domain, form, langs, current_lan
...
+        if not has_case_error:
```
Comment (kaapstorm) [inline corehq/apps/app_manager/views/forms.py]: Don't you mean `if has_case_error`?

Comment (kaapstorm) [reply corehq/apps/app_manager/views/forms.py]: Oh! Sorry I misread the context.

**Review by kaapstorm — APPROVED** [review]

File: corehq/apps/app_manager/views/forms.py (hunk 2 above)
Comment (orangejenny) [reply corehq/apps/app_manager/views/forms.py]: no - if `has_case_error` is true, this message will already have been a message added

Comment (orangejenny) [reply corehq/apps/app_manager/views/forms.py]: jinx :)

**Pull request merged**